docs/os-ref.md

```diff
@@ -293,25 +293,12 @@ Parameters allow the ACL to perform certain computations with the arguments of a
-Even though `uint240`s are used, it can be used to store any integer up to `2^30 - 1`, addresses, and bytes32. In the case of comparing hashes, losing 2 bytes of precision shouldn't be a dealbreaker if the hash algorithm is secure.
-- **Argument ID** (`uint8`): Determines how the comparison value is fetched. From 0 to 200 it refers to the argument index number passed to the role. After 200, there are some special *Argument IDs*:
-	- `BLOCK_NUMBER_PARAM_ID` (`id = 200`): sets comparison value to the block number at the time of execution. This allows for setting up timelocks depending on blocks.
-	- `TIMESTAMP_PARAM_ID` (`id = 201`): sets comparison value to the timestamp of the current block at the time of execution. This allows for setting up timelocks on time.
-	- `id = 202`: not currently in use.
-	- `ORACLE_PARAM_ID` (`id = 203`): checks with an oracle at the address in the `argument value` and returns whether it returned true or false (no comparison with the `argument value`).
-	- `LOGIC_OP_PARAM_ID` (`id = 204`): evaluates a logical operation and returns true or false depending on its result (no comparison with the `argument value`).
-	- `PARAM_VALUE_PARAM_ID` (`id = 205`): return `argument value`. Commonly used with the `RET` operation to just return a value. If the value in the param is greater than 0 it will evaluate to true, otherwise it will return false.
-- **Operation type** (`uint8`): what operation should be done to compare the value fetched using the argument ID or the argument value. For all comparisons, both values are compared in the following order `args[param.id] <param.op> param.value`. Therefore, for a greater than operation, with `param = {id: 0, op: Op.GT, value: 10}`, it will interpret whether the argument 0 is greater than 10. The implemented operation types are:
-	- None (`Op.NONE`): always evaluates to `false` regardless of parameter or arguments.
-	- Equals (`Op.EQ`): evaluates to true if every byte matches between `args[param.id]` and `param.value`.
-	- Not equals (`Op.NEQ`): evaluates to true if any byte doesn't match.
-	- Greater than (`Op.GT`): evaluates to true if `args[param.id] > param.value`.
-	- Less than (`Op.LT`): evaluates to true if `args[param.id] < param.value`.
-	- Greater than or equal (`Op.GTE`): evaluates to true if `args[param.id] >= param.value`.
-	- Less than or equal (`Op.LTE`): evaluates to true if `args[param.id] <= param.value`.
-	- Return (`Op.RET`): evaluates to true if `args[param.id]` is greater than one. Used with `PARAM_VALUE_PARAM_ID`, it makes `args[param.id] = param.value`, so it returns the associated value of the parameter.
+  Even though `uint240`s are used, it can be used to store any integer up to `2^30 - 1`, addresses, and bytes32. In the case of comparing hashes, losing 2 bytes of precision shouldn't be a dealbreaker if the hash algorithm is secure.
```

Comment:
This bit looks a bit weird on the markdown renderer, and needs the whitespace for the bullet points. Was there anything you added to this section (otherwise can we revert it)?

Reply:
I have a markdown linter setup on vscode. I believe that's why 😅